codegen/src/api/calls.rs

```diff
@@ -78,6 +78,10 @@ pub fn generate_calls(
             let function_name = struct_def.name.to_string().to_snake_case();
             let fn_name = format_ident!("{}", function_name);
 
+            // Propagate the documentation just to `TransactionApi` methods, while
```

Comment:
can you explain the why the docs for the inner call structures are "drained" and generated for the wrappers only?

Reply:
My initial intuition was to keep the documentation on both the calling methods and inner structures.
The user should call the exposed API and not necessarily care about the inner structures used to construct the call. 

```rust
  pub struct Transfer {
   ...
      pub value: ::core::primitive::u128,
  }

  /// Doc goes here
  pub fn transfer( ... ) {
      let call = Transfer { dest, value };
      ::subxt::SubmittableExtrinsic::new(self.client, call)
  }
```

Although someone could potentially submit a manually constructed extrinsic (as the `pub fn transfer` implementation), I think this is quite uncommon. 

Would you think having documentation on both would also satisfy those manually constructed cases? 🤔

Reply:
Ok, I agree the outer docs should be sufficient then.

I think someone constructs their own extrinsics I guess they are on their own anyway.